cmd/neofs-cli/modules/object.go

```diff
@@ -869,29 +887,29 @@ func parseObjectNotifications(cmd *cobra.Command) (*object.NotificationInfo, err
 }
 
 func getCID(cmd *cobra.Command) (*cid.ID, error) {
-	id := cid.New()
+	var id cid.ID
 
-	err := id.Parse(cmd.Flag("cid").Value.String())
+	err := id.DecodeString(cmd.Flag("cid").Value.String())
 	if err != nil {
 		return nil, fmt.Errorf("could not parse container ID: %w", err)
 	}
 
-	return id, nil
+	return &id, nil
 }
 
 func getOID(cmd *cobra.Command) (*oidSDK.ID, error) {
```

Comment:
There is a number of places where we still use pointers, do you want to fix them in this PR or is it too much work?

Reply:
I'd like to resolve pointer theme globally in a separate issue/PR.